routesrv/testdata/lb-target-multi.eskip

```diff
@@ -0,0 +1,15 @@
+// default backend, target 1:
+kube_namespace1__ingress1______:
+  *
+  -> <roundRobin, "http://42.0.1.2:8080", "http://42.0.1.3:8080">;
```

Comment:
I think we should add another test that has no default backend.
The normal case for us at least is, that we have no default backend.

Reply:
For this one and one below: My assumption was that the underlying data client is tested thoroughly already in other packages and the purpose of this test suite is to test `routesrv`. So I actually copied those two fixtures kinda randomly, because there were small enough and simply different. I might have chosen completely different ones, because I just needed some to feed to the test API server. Do you think we should test the portions of the Kubernetes data client behavior here as well?